metadata-ws/src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/Taxonomy.java

```diff
@@ -84,8 +91,10 @@ public String getName() {
      * Release date control: taxonomies are always public.
      */
     @Override
+    @JsonIgnore
     public LocalDate getReleaseDate() {
         return null;
     }
 
+    public enum Rank {SUBSPECIES, SPECIES, GENUS, ORDER, CLASS}
```

Comment:
Following my comments above, I think that subspecies should be excluded from this list because it is rarely used and is definitely not necessary for indexing